chain/client-primitives/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "near-client-primitives"
+version = "0.1.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix = "0.9"
+chrono = { version = "0.4.4", features = ["serde"] }
+serde = { version = "1", features = ["derive"] }
+strum = { version = "0.18", features = ["derive"] }
+
+near-chain = { path = "../chain" }
+near-chain-primitives = { path = "../chain-primitives" }
+near-chunks = { path = "../chunks" }
+near-network = { path = "../network" }
```

Comment:
Ideally, we don't want to depend on non-primitive dependencies.

Can you review the usage and consider moving certain parts around (e.g. if we moved something unnecessary to the *-primitives, and would be better to leave it in the original place)?